experiments/conductor/cmd/runner/runner.go

```diff
@@ -82,6 +82,8 @@ func BuildRunnerCmd() *cobra.Command {
 		"", "", "dedicated directory for logging, empty for stdout.")
 	cmd.Flags().DurationVarP(&opts.timeout, "timeout",
 		"t", 5*time.Minute, "Global timeout for commands.")
+	cmd.Flags().IntVarP(&opts.defaultRetries, "retries",
+		"r", 10, "Default number of retries for failed commands.")
```

Comment:
Nit: We should suggest increasing timeout when the number of retries here is large.